analysers/analyser_merge_parapharmacy_IT.py

```diff
@@ -27,13 +27,13 @@
 class Analyser_Merge_Parapharmacy_IT(Analyser_Merge):
     def __init__(self, config, logger = None):
         Analyser_Merge.__init__(self, config, logger)
-        self.missing_official = self.def_class(item = 7150, id = 21, level = 3, tags = ['merge', 'highway'],
+        self.missing_official = self.def_class(item = 8210, id = 21, level = 3, tags = ['merge', 'highway'],
             title = T_('Pharmacy not integrated'))
-        self.missing_osm      = self.def_class(item = 7250, id = 22, level = 3, tags = ['merge', 'highway'],
+        self.missing_osm      = self.def_class(item = 8212, id = 22, level = 3, tags = ['merge', 'highway'],
```

Comment:
No, `missing_osm` should not chnage and stay at `7250`

Reply:
done